nodemon.json

```diff
@@ -6,11 +6,10 @@
     "config/",
     "locales/",
     "server.js",
-    "start.js"
-  ],
-  "ignore": [
-    ".build/"
+    "start.js",
+    ".build/frameworks/**/*.json"
```

Comment:
Is it worth adding `.env` too?

Reply:
Not sure just restarting the server would be enough would it? Wouldn't you need to reload you shell session too?